src/App.js

```diff
@@ -1,12 +1,73 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import Pokemon from './components/Pokemon';
 import './App.css';
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [data, setData] = useState([]);
```

Comment:
Ik zou de default waarde aanpassen naar undefined, zodat je een beter onderscheid hebt tussen of iets wél of niet is ingeladen
```suggestion
    const [data, setData] = useState();
```